source/extensions/omni.isaac.lab_tasks/omni/isaac/lab_tasks/manager_based/locomotion/velocity/velocity_env_cfg.py

```diff
@@ -172,6 +172,15 @@ class EventCfg:
             "operation": "add",
         },
     )
+    
+    com_randomize = EventTerm(
+        func=mdp.randomize_body_coms,
+        mode="startup",
+        params={
+            "asset_cfg": SceneEntityCfg("robot", body_names="trunk"),
```

Comment:
Not all robots have bodies named trunk. In the default configuration, it is called "base" and we set it to "trunk" wherever applicable (inside Unitree Go1, Go2, etc configs).